api/client/options.proto

```diff
@@ -20,6 +20,8 @@ message RequestOptions {
   // Our StreamDecoder depends on bounding the size here, so if this changes, an amendment
   // to that is needed as well.
   google.protobuf.UInt32Value request_body_size = 3 [(validate.rules).uint32 = {lte: 4194304}];
+  // New field for customizable JSON request body
```

Comment:
I would remove New field from the comment. How about "Customizable JSON request body"?